docroot/modules/custom/civictheme_migrate/src/Utils/TextHelper.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Drupal\civictheme_migrate\Utils;
+
+use Drupal\Component\Utility\Html;
+use Drupal\file\Entity\File;
+
+/**
+ * Text Support helper class.
+ *
+ * @package Drupal\civictheme_migrate\Utils
+ */
+class TextHelper {
+
+  /**
+   * Remove whitespaces from a string.
+   *
+   * @param string $value
+   *   The string.
+   *
+   * @return string
+   *   The clean string.
+   */
+  public static function trimWhitespaces(string $value) : string {
+    $value = preg_replace("/(\n|\t|\r)/", '', $value);
+    $value = preg_replace('/ {2,}/', ' ', $value);
+    $value = preg_replace('/[\x{200B}-\x{200D}]/u', '', $value);
+    return trim($value);
+  }
+
+  /**
+   * Convert <img> tags into Drupal embedded media entities.
+   *
+   * @param string $text
+   *   The inout text.
+   *
+   * @return string
+   *   The text.
+   */
+  public static function convertEmbeddedImagesWithMedia(string $text) : string {
```

Comment:
@govindmaloo 
please add Kernel tests for this file

Reply:
Added the Required kernel Test